docs/running-rules.md

```diff
@@ -9,39 +9,39 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in open-source Semgrep Registry that enables you to scan code without need to write anything custom.
+Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. The list below covers different kinds of Semgrep rules:
 
-Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>.
+- Use existing [Semgrep Registry rules](#running-semgrep-registry-rules) or add new rules to Semgrep Registry and run them. See [Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/) to find out how to add new rules.
+- Local rules:
+  - Create and run one-off [ephemeral rules](#ephemeral-rules) in the command line.
+  - Create and run [YAML-defined rules in a file](#creating-and-using-yaml-defined-rules-file).
+- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
 
-<div className="lang-container" style={{marginBottom: '20px'}}>
-  <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
-</div>
+You can run all rules on your local code or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI. For more information, see [Semgrep CI overview](semgrep-ci/overview.md).
 
-You can run existing and custom Semgrep rules locally with the Semrgep command line tool or continuously with Semgrep CI. See [Getting started](../getting-started/) for their respective installation and setup.
+## Running Semgrep Registry rules locally
 
-## Run Registry rules
-
-Explore the [Semgrep Registry](https://semgrep.dev/explore) and run rules and rulesets via:
-
-```sh
-# Automatically survey languages and frameworks and run recommended Registry rules
-semgrep --config=auto path/to/src
+Try to run the following example of [Semgrep Registry](https://semgrep.dev/explore) rules:
 
-# Run a ruleset with rules for many languages using --config
-semgrep --config=<ruleset-id> path/to/src
-
-# Run simultaneously with Registry rulesets and local rules
-semgrep --config=<ruleset-id> --config=path/to/yml path/to/src
-```
+1. Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code:
```

Comment:
Number (1) and Number (2) are two different procedures. (1) Is a single step procedure, since all they need to run semgrep is to run `semgrep --config auto`

Should be:

A. Test how Semgrep automatically surveys...
B. Explore the Semgrep Registry and choose a rule..
 B.1 On the page of the rule, click **Run Locally**...
 ... up until B.4 Optional

Reply:
Sure!